### PR TITLE
A bunch of gridlist fixes

### PR DESCRIPTION
**Pull request by shyndman**

### CODE REVIEW

**Comment by ThomasBurleson** [comment]
@shyndman - the demo #3: Responsive Usage does not appear to be working correctly.

screen shot 2015-04-06 at 1 08 19 pm

File: src/components/gridList/demoDynamicTiles/index.html [hunk 1]
```diff
@@ -1,18 +1,17 @@
 <div ng-app="gridListDemo" ng-controller="gridListDemoCtrl as vm" flex>
+  <md-button ng-click="vm.rotate()">Rotate</md-button>
```
Comment (ThomasBurleson) [inline src/components/gridList/demoDynamicTiles/index.html]: I think we either need to:
*  put this below the Grid or 
*  use a FAB in the TopLeft corner of the Grid.

**Comment by ThomasBurleson** [comment]
Now it appears that all demos are broken when I attempt to merge this PR.

**Comment by shyndman** [comment]
See the problem. Give me a few.

**Comment by shyndman** [comment]
Got it. Grid wasn't being resized properly.

Also rebased on latest master.

File: src/components/gridList/demoDynamicTiles/index.html (hunk 1 above)
Comment (shyndman) [inline src/components/gridList/demoDynamicTiles/index.html]: Might be easier for me to ditch the button. It was useful for testing, but I'm not sure it offers much to the demo.

Comment (ThomasBurleson) [inline src/components/gridList/demoDynamicTiles/index.html]: :+1:

**Pull request closed without merging**